concurrency-limits-core/build.gradle

```diff
@@ -5,10 +5,10 @@ plugins {
 sourceCompatibility = JavaVersion.VERSION_1_8
 
 dependencies {
-    implementation "org.slf4j:slf4j-api:1.7.+"
-    
-    testImplementation 'junit:junit-dep:4.10'
-    testImplementation "org.slf4j:slf4j-log4j12:1.7.+"
+    implementation "org.slf4j:slf4j-api:${slf4jVersion}"
 
-    testImplementation 'com.github.kevinmost:junit-retry-rule:cbdd972d7c'
+    testCompileOnly "junit:junit:${jUnitLegacyVersion}"
```

Comment:
You might want to leverage JUnit's BOM
https://junit.org/junit5/docs/current/user-guide/#dependency-metadata-junit-bom
e.g.
```
testImplementation(enforcedPlatform("org.junit:junit-bom:5.9.0"))
```